exporter/zipkin/zipkin.go

```diff
@@ -50,6 +54,36 @@ func NewExporter(reporter reporter.Reporter, localEndpoint *model.Endpoint) *Exp
 	}
 }
 
+type Options struct {
+	// EndpointHostPort is the host:port combination for the Zipkin service
+	// e.g. 192.168.0.2:8080
+	EndpointHostPort string
```

Comment:
Please not, it is already too many steps to configure an exporter.

s/EndpointHostPort/Endpoint

Reply:
Although, Endpoint requires the host:port string, should I then add a comment about Endpoint as alluded to in https://github.com/census-instrumentation/opencensus-go/pull/453#discussion_r168641402?

Reply:
Give an example and it will be enough.